From 06317f4f6a30ea6c141e8e0e65c5e65628799e36 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Fri, 10 May 2019 23:17:55 +0200
Subject: [PATCH 011/152] status/keyboard: Make Input source types var

These are used by keyboard so they need to be readable outside the
status.keyboard module.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/700
---
 js/ui/status/keyboard.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/js/ui/status/keyboard.js b/js/ui/status/keyboard.js
index dbe5ded51..9383e9c2c 100644
--- a/js/ui/status/keyboard.js
+++ b/js/ui/status/keyboard.js
@@ -13,8 +13,8 @@ const PanelMenu = imports.ui.panelMenu;
 const SwitcherPopup = imports.ui.switcherPopup;
 const Util = imports.misc.util;
 
-const INPUT_SOURCE_TYPE_XKB = 'xkb';
-const INPUT_SOURCE_TYPE_IBUS = 'ibus';
+var INPUT_SOURCE_TYPE_XKB = 'xkb';
+var INPUT_SOURCE_TYPE_IBUS = 'ibus';
 
 var LayoutMenuItem = GObject.registerClass(
 class LayoutMenuItem extends PopupMenu.PopupBaseMenuItem {
-- 
2.23.0

